app/src/main/java/com/example/parkingsystem/database/ParkingReservationDatabase.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.parkingsystem.database
+
+import com.example.parkingsystem.entity.ParkingLotReservation
+import com.example.parkingsystem.util.Constants
+
+object ParkingReservationDatabase {
+    val hashMapReservation: MutableMap<Int, MutableList<ParkingLotReservation>> = HashMap()
+    private var parkingLotSize: Int = Constants.ZERO
+
+    fun addReservation(reservation: ParkingLotReservation) {
+        var reservations: MutableList<ParkingLotReservation>? = mutableListOf()
```

Comment:
It's not necessary make this variable nullable. remove the "?".

Reply:
Done